packages/css/src/components/accordion/README.md

```diff
@@ -1,42 +1,49 @@
-| Shift + Tab          | Ga naar het volgende element dat focus kan hebben                                                                            |
-| Pijltje naar beneden | Als een accordionsectie focus heeft: ga naar de volgende sectie. Als de laatste sectie focus heeft, ga naar de eerste sectie |
-| Pijltje omhoog       | Als een accordionsectie focus heeft: ga naar de vorige sectie. Als de eerste sectie focus heeft, ga naar de laatste sectie   |
-| Home                 | Als een accordionsectie focus heeft: ga naar de eerste sectie                                                                |
-| End                  | Als een accordionsectie focus heeft: ga naar de laatste sectie                                                               |
-
-## Relevante WCAG eisen
-
-De WCAG eisen voor het Button component en het Heading component gelden ook voor de accordion header.
-
-Let extra op voor deze onderdelen:
-
-- [WCAG eis 1.3.1](https://www.w3.org/TR/WCAG21/#info-and-relationships): de _heading level_ van de accordionsecties is afhankelijk van waar in de pagina de accordion is geplaatst, dit kan per pagina verschillen.
-- [WCAG eis 1.3.6](https://www.w3.org/TR/WCAG21/#identify-purpose): gebruik `aria-controls` voor de button, en gebruik een `region` landmark voor de _expandable region_ (gebruik daarvoor het HTML-element `<section>`)
-- [WCAG eis 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum): contrast tussen tekst en achtergrond en tussen icoon en achtergrond is voldoende in alle varianten, alle interactieve statussen en alle mogelijke combinaties.
-- [WCAG eis 3.2.1](https://www.w3.org/TR/WCAG21/#on-focus): maak de accordion niet automatisch _expanded_ als de button focus krijgt.
-- [WCAG eis 2.1.3](https://www.w3.org/TR/WCAG21/#keyboard-no-exception): ondersteun ook de optionele toetsen: `Down Arrow`, `Up Arrow`, `Home` en `End`. `Space` moet de button activeren, niet de pagina scrollen (gebruik `keypressEvt.preventDefault()`).
-- [WCAG eis 2.4.6](https://www.w3.org/TR/WCAG21/#headings-and-labels): de tekst die zowel wordt gebruikt als heading en als label voor de button moet duidelijk zijn, omdat de inhoud niet altijd zichtbaar is.
-- [WCAG eis 2.4.10](https://www.w3.org/TR/WCAG21/#section-headings): accordions die onderdeel vormen van de lopende tekst moeten section headings gebruiken.
-
-## Referenties
+The accordion component is a clickable (vertically stacked) list of headings to hide or show associated content.
+With an accordion, you offer content to users progressively.
+
+## Guidelines
+
+- Use an accordion on a full page with essential and non-essential content.
+- Hiding non-essential content helps users get to important content more quickly.
+- Avoid using an accordion to avoid too much essential content on 1 page.
+- Rewrite the page more compactly, divide the content over multiple pages or use the (yet to be created) table of contents component instead of an accordion for that purpose.
```

Comment:
I would comment the entire line with a `@TODO` instead of using "Yet to be created" in the content. Or perhaps make note of the file and line number in the Jira story of ToC.